assets/js/admin-scripts.js

```diff
@@ -2158,3 +2167,38 @@ function edd_attach_tooltips( selector ) {
 		}
 	});
 }
+
+function easy_digital_downloads_sendwp_remote_install() {
+	var data = {
+		'action': 'easy_digital_downloads_sendwp_remote_install',
+	};
+
+	// since 2.8 ajaxurl is always defined in the admin header and points to admin-ajax.php
+	jQuery.post(ajaxurl, data, function(response) {
+		var data = JSON.parse(response);
+		easy_digital_downloads_sendwp_register_client(data.register_url, data.client_name, data.client_secret, data.client_redirect, data.partner_id);
```

Comment:
This doesn't check for failures in `wp_ajax_easy_digital_downloads_sendwp_remote_install` response and can cause the registration redirection to be invalid/incorrect.

Reply:
Added simple handling for it.